spec/sail_spec.vim

```diff
@@ -11,7 +11,7 @@ describe "Laravel Sail"
 
   after
     call Teardown()
-    !rm -rf vendor
+    !rm -rf vendor docker-compose.yml
```

Comment:
Running the testsuite locally, this file was not cleaned up. Just making sure we have a fresh slate for every test.